### ISSUE TITLE
Solving conflicts

### ISSUE THREAD

**Issue by DamienCassou** [pimutils/vdirsyncer#642]
If there is a conflict, I don't know of any way to resolve it. I would like to be able to say, for each conflict, things like:

- show me the difference,
- take side A (or side B)
- open both and let me do the merge

Related: #521

**Comment by untitaker** [comment]
There are multiple options documented at
https://vdirsyncer.pimutils.org/en/stable/tutorial.html#conflict-resolution and https://vdirsyncer.pimutils.org/en/stable/config.html#pair-section

**Comment by DamienCassou** [comment]
I already read the whole manual some times ago and read the conflict-resolution section yesterday too. Thanks to your second url, I've just learned about the `"command"` option that's exactly what I wanted (what about referencing that in the conflict resolution section? Want a PR?). I wrote a script (see below) that works well.

My problem is that the 2 files are so different (one is modified by khard, the other by davdroid) that it's hard to merge the changes. One way to simplify merging would be to use khard on both files to generate two other, easier to merge files (what do you think @scheibler?).

Here is the script I put in there. It seems to work fine (it uses Emacs and its conflict solving facilities):

```bash
#!/usr/bin/env bash

file1="$1"
file2="$2"

if [[ ! -f $file1 || ! -f $file2 ]]; then
    echo "ERROR: Wrong call $*"
    echo "Usage: $0 FILE1 FILE2"
    exit 1
fi

target=$(mktemp)

emacs-client.sh --eval "(ediff3 \"$file1\" \"$file2\" \"$target\")"

echo "Merge these two files into ${target}"
read -p "Press RET here when you are done (C-c to abort)... "

if [[ ! -s $target ]]; then
    echo "ERROR: The $target is non-existent or empty"
    exit 1
fi

cp "$target" "$file1"
cp "$target" "$file2"

echo "The merge went well, going on."
```

**Comment by untitaker** [comment]
>(what about referencing that in the conflict resolution section? Want a PR?

If you have ideas how to improve the tutorials, of course PRs are welcome. Note that the manual underwent some changes which will be released with 0.16. Take a look at https://vdirsyncer.pimutils.org/en/latest/tutorial.html#conflict-resolution (latest in the URL instead of stable)

I do want to normalize files before sending them to conflict resolution (see https://github.com/pimutils/vdirsyncer/issues/521), but with the current "knowledge" vdirsyncer has of the item content, that's not possible. Perhaps one day.

I'll close this for now though, again, PRs regarding mentioning this in the docs are welcome, for your other point there is already a fitting issue.
